constructor/templates/constructor/update.html

```diff
@@ -15,11 +15,16 @@
             <form method="POST">
                 {% csrf_token %}
                 <!--prevents any csrf attacks-->
+                <div class="inputtitle">
+                    {{form.title}}
+                </div>
                 {% for field in form %}
-                    <br>
-                    <div class="mydiv">
-                        {{ field }}
-                    </div>
+                    {% if forloop.counter > 1 %}
+                        <br>
+                        <div class="mydiv">
```

Comment:
What elements of form do you skip?

Reply:
Title, because it is already included in form before